src/routes/index.ts

```diff
@@ -18,5 +18,10 @@
  */
 export enum ROUTES {
```

Comment:
Why `enum` instead of `const`?

Reply:
It's a TypeScript thing, but yeah, `const` would pretty much serve the same purpose. Benefit of `enum` is that TS knows it's a more strict enumeration of all possible categories. If you're looking to create an encyclopedia of constants, it's normally a good way to go. (Caveat: I only think that's string of "string enums", like you see here. I actually dislike the standard issue TS enum, I find it confusing when you're trying to debug and too prone to errors in usage.)

https://www.typescriptlang.org/docs/handbook/enums.html#string-enums

Reply:
Noted, thanks!